docker-rsync-local-leonardo.sh

```diff
@@ -0,0 +1,137 @@
+
+echo "Creating shared volumes if they don't exist..."
+docker volume create --name leonardo-shared-source
+docker volume create --name jar-cache
+
+echo "Launching rsync container..."
+docker run -d \
+    --name leonardo-rsync-container \
+    -v leonardo-shared-source:/working \
+    -e DAEMON=docker \
+    tjamet/rsync
+
+run_rsync ()  {
+    rsync --blocking-io -azl --delete -e "docker exec -i" . leonardo-rsync-container:working \
+        --filter='+ /build.sbt' \
+        --filter='+ /http/' \
+        --filter='+ /http/src/***' \
+        --filter='+ /core/' \
+        --filter='+ /core/src/***' \
+        --filter='+ /config/***' \
+        --filter='+ /docker/***' \
+        --filter='+ /jupyter-docker/***' \
+        --filter='+ /project/***' \
+        --filter='+ /.git/***' \
+        --filter='- *'
+}
+echo "Performing initial file sync..."
+run_rsync
+fswatch -o . | while read f; do run_rsync; done &
+echo $$ > ./.docker-rsync-local.pid
+
+start_server () {
+    docker network create fc-leonardo
+
+    echo "Creating Google sqlproxy container..."
+    docker create --name sqlproxy \
+    --restart "always" \
+    --network="fc-leonardo" \
+    --env-file="./config/sqlproxy.env" \
+    broadinstitute/cloudsqlproxy:1.11_20180808
+
+    docker cp config/sqlproxy-service-account.json sqlproxy:/etc/sqlproxy-service-account.json
+
+    echo "Creating helm docker container..."
+    docker build -t helm-lib - < $HELM_SCALA_SDK_DIR/Dockerfile
+    docker create --name leonardo-helm-lib -v /build \
+    helm-lib
+
+    echo "Creating SBT docker container..."
+    docker create -it --name leonardo-sbt \
+    -v leonardo-shared-source:/app -w /app \
+    -v jar-cache:/root/.ivy -v jar-cache:/root/.ivy2 \
+    -p 25050:5050 \
+    --network=fc-leonardo \
+    --env-file="env/local.env" \
+    --env-file="rendered/secrets.env" \
+    -e HOSTNAME=$HOSTNAME \
+    -e JAVA_OPTS='-DXmx4G -DXms4G -DXss2M -Dsun.net.spi.nameservice.provider.1=default -Dsun.net.spi.nameservice.provider.2=dns,Jupyter -Djna.library.path=/helm-go-lib-build' \
+    hseeberger/scala-sbt:17.0.2_1.6.2_3.1.1 \
+    sbt http/run
+
+    echo "Copying files to SBT docker container..."
+    docker cp config/leonardo-account.pem leonardo-sbt:/etc/leonardo-account.pem
+    docker cp config/leonardo-account.json leonardo-sbt:/etc/leonardo-account.json
+    docker cp config/jupyter-server.crt leonardo-sbt:/etc/jupyter-server.crt
+    docker cp config/jupyter-server.key leonardo-sbt:/etc/jupyter-server.key
+    docker cp config/leo-client.p12 leonardo-sbt:/etc/leo-client.p12
+    docker cp config/rootCA.key leonardo-sbt:/etc/rootCA.key
+    docker cp config/rootCA.pem leonardo-sbt:/etc/rootCA.pem
+    docker cp /etc/localtime leonardo-sbt:/etc/localtime
+    docker cp leonardo-helm-lib:/build /tmp
+    docker cp /tmp/build/ leonardo-sbt:/helm-go-lib-build
+#    docker cp env/local.env leonardo-sbt:/etc/local.env
+#    dokcer cp rendered/secrets.env leonardo-sbt:/etc/secrets.env
```

Comment:
typo. also, are these working/testing things that should be removed before commit or switches for special dev customization